### PR TITLE
[New Rule] - Executable File Created by a System Critical Process

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/182

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_system_critical_proc_abnormal_file_activity.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/08/19"
+ecs_version = ["1.5.0"]
```
Comment (bm11100) [inline rules/windows/defense_evasion_system_critical_proc_abnormal_file_activity.toml:3]:
```suggestion
ecs_version = ["1.6.0"]
```

**Review by bm11100 — APPROVED** [review]
LGTM

File: rules/windows/defense_evasion_system_critical_proc_abnormal_file_activity.toml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+name = "Unusual Executable File Creation by a System Critical Process"
+risk_score = 73
+rule_id = "e94262f2-c1e9-4d3f-a907-aeab16712e1a"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:file and not event.type:deletion and
+  file.name:(*.exe or *.dll) and
```
Comment (rw-access) [inline rules/windows/defense_evasion_system_critical_proc_abnormal_file_activity.toml:25]: file.extension instead

**Review by rw-access — CHANGES_REQUESTED** [review]
I think we should switch to file.extension instead. It's much more efficient than a leading `*`

File: rules/windows/defense_evasion_system_critical_proc_abnormal_file_activity.toml (hunk 2 above)
Comment (Samirbous) [reply rules/windows/defense_evasion_system_critical_proc_abnormal_file_activity.toml]: changed, rule was created before file.extension issue was raised

File: rules/windows/defense_evasion_system_critical_proc_abnormal_file_activity.toml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+license = "Elastic License"
+name = "Unusual Executable File Creation by a System Critical Process"
+risk_score = 73
+rule_id = "e94262f2-c1e9-4d3f-a907-aeab16712e1a"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:file and not event.type:deletion and
```
Comment (brokensound77) [inline rules/windows/defense_evasion_system_critical_proc_abnormal_file_activity.toml:24]:
```suggestion
event.category:process and event.type:(start or process_started) and
```

**Review by brokensound77 — APPROVED** [review]

**Pull request merged**